eca-chat.el

```diff
@@ -26,16 +26,38 @@
   :type 'hook
   :group 'eca)
 
-(defcustom eca-chat-window-width 0.35
-  "The width of `eca' dedicated chat window."
+(defcustom eca-chat-window-side 'right
+  "Side of the frame where the ECA chat window should appear.
+Can be `'left', `'right', `'top', or `'bottom'.  This setting will only
+be used when `eca-chat-use-side-window' is non-nil."
+  :type '(choice (const :tag "Left" left)
+                 (const :tag "Right" right)
+                 (const :tag "Top" top)
+                 (const :tag "Bottom" bottom))
+  :group 'eca)
+
+(defcustom eca-chat-window-width 90
```

Comment:
Just tested locally and the width is huge, I think we can go with the old width + a little bit more, like 0.40.
```suggestion
(defcustom eca-chat-window-width 0.40
```